service/importer/service_file.go

```diff
@@ -105,6 +105,15 @@ func newValidator() (*validator.Validate, ut.Translator) {
 	uni := ut.New(en, en)
 	trans, _ := uni.GetTranslator("en")
 	validate := validator.New()
+
+	validate.RegisterValidation("env", xvalidator.IsEnv)
+	validate.RegisterTranslation("env", trans, func(ut ut.Translator) error {
+		return ut.Add("portmap", "{0} must be a valid env variable name", false)
```

Comment:
why `portmap` here ?
```suggestion
		return ut.Add("env", "{0} must be a valid env variable name", false)
```